accepted/PSR-15-request-handlers.md

```diff
@@ -65,7 +65,7 @@ specific HTTP message implementation.
 
 ### 1.4 Handling Exceptions
 
-It is RECOMMENDED that any application using middleware include a component
+It is RECOMMENDED that any application using middleware includes a component
```

Comment:
This doesn't seem grammatically correct me.

Reply:
English being the hodge podge of weirdness that it is, "application includes" or "applications include".  One or the other but not both gets an S.  (It's a XOR operation.)

Don't ask me, I don't understand this wonky language we use...

Reply:
This one is straightforward... subject+verb is application+includes... both are singular.  A plural-ish prepositional phrase between the two is the usual blame for getting the verb wrong.

Reply:
The real reason behind this is context. The next sentence uses singular and not plural. Was just for cohesion.